Xamarin.Forms.Core/Cells/Cell.cs

```diff
@@ -12,7 +12,8 @@ namespace Xamarin.Forms
 	public abstract class Cell : Element, ICellController, IFlowDirectionController, IPropertyPropagationController, IVisualController
 	{
 		public const int DefaultCellHeight = 40;
-		public static readonly BindableProperty IsEnabledProperty = BindableProperty.Create("IsEnabled", typeof(bool), typeof(Cell), true, propertyChanged: OnIsEnabledPropertyChanged);
+		public static readonly BindableProperty IsEnabledProperty = BindableProperty.Create(nameof(IsEnabled), typeof(bool), typeof(Cell), true, propertyChanged: OnIsEnabledPropertyChanged);
```

Comment:
It would be interesting to give some kind of visual feedback when disabling the cell. Maybe a new property to define the color when disabled.